src/main/scala/org/dia/apps/MCSApp.scala

```diff
@@ -50,6 +50,14 @@ object MCSApp extends App {
   val minAreaThres = 16
   val minGraphLength = 4
 
+  /** Criteria for MCC */
+  val minFeatureLength = 6
```

Comment:
For the purposes of not adding another dataset to test, make this length 5 instead of 6.